sway-core/src/semantic_analysis/namespace/module.rs

```diff
@@ -41,6 +41,8 @@ pub struct Module {
     pub(crate) submodules: im::OrdMap<ModuleName, Module>,
     /// The set of symbols, implementations, synonyms and aliases present within this module.
     items: Items,
+    /// Name of the module, package name for root module, dependency name for other modules
```

Comment:
Would you be able to add some examples of what this would look like in the rustdoc comment here? I'm kind of struggling a little bit to solidify what this field is going to mean concretely.